.gitignore

```diff
@@ -0,0 +1,5 @@
+.eggs/
```

Comment:
Here's the default template we use:

https://github.com/zopefoundation/meta/blob/master/config/default/gitignore.j2